bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -122,70 +128,128 @@ async def autoreview_loop(self) -> None:
+
+            await nomination_discussion.send(
+                f":warning: <@{nomination.user_id}> ({nomination.user_id})"
+                " was removed from the talentpool due to inactivity."
+            )
+            await self.api.edit_nomination(
+                nomination.id,
+                active=False,
+                end_reason=f"Automatic removal: User was inactive for more than {DAYS_UNTIL_INACTIVE}"
+            )
+
+    @nomination_group.group(
+        name="list",
+        aliases=("nominated", "nominees"),
+        invoke_without_command=True
     )
     @has_any_role(*MODERATION_ROLES)
-    async def list_command(
+    async def list_group(
         self,
         ctx: Context,
-        oldest_first: bool = False,
     ) -> None:
         """
-        Shows the users that are currently in the talent pool.
+        Shows the users that are currently in the talent pool, ordered by next to be reviewed.
 
-        The optional kwarg `oldest_first` can be used to order the list by oldest nomination.
+        Note that this order will change over time, so should not be relied upon.
         """
-        await self.list_nominated_users(ctx, oldest_first=oldest_first)
+        await self.list_nominated_users(ctx, order_by_priority=True)
+
+    @list_group.command(name="oldest")
+    async def list_oldest(self, ctx: Context) -> None:
+        """Shows the users that are currently in the talent pool, ordered by oldest nomination."""
+        await self.list_nominated_users(ctx, oldest_first=True)
+
+    @list_group.command(name='newest')
+    async def list_newest(self, ctx: Context) -> None:
+        """Shows the users that are currently in the talent pool, ordered by newest nomination."""
+        await self.list_nominated_users(ctx, oldest_first=False)
 
     async def list_nominated_users(
         self,
         ctx: Context,
         oldest_first: bool = False,
+        order_by_priority: bool = False
     ) -> None:
         """
-        Gives an overview of the nominated users list.
-
-        It specifies the user's mention, name, how long ago they were nominated, and whether their
-        review was posted.
+        Lists the currently nominated users.
 
         The optional kwarg `oldest_first` orders the list by oldest entry.
         """
+        now = datetime.now(tz=timezone.utc)
+        embed = Embed(
+            title="Talent Pool active nominations",
+            color=Color.blue()
+        )
         nominations = await self.api.get_nominations(active=True)
-        if oldest_first:
-            nominations = reversed(nominations)
 
-        lines = []
+        if not nominations:
+            embed.description = "There are no active nominations."
+            await ctx.send(embed=embed)
+            return
+
+        if order_by_priority:
+            nominations = await self.reviewer.sort_nominations_to_review(nominations, now)
+        elif oldest_first:
+            nominations.reverse()
```

Comment:
Given the names of the kwargs it wouldn't make sense to pass both as `True`. 

I can't think of a better way of naming the options without adding e.g. an enum for criteria and order or splitting it into multiple functions, which feels a bit overkill given this works for the current case.

Reply:
Hmm, I don't see the issue. You can order by priority, just in reverse.

Reply:
Not a big deal though, since this doesn't affect any planned feature. Your call.

Reply:
What about having an `NominationOrdering` enum, that would have the `priority` and `nomination_date` or sthg within those lines ?

Reply:
> Hmm, I don't see the issue. You can order by priority, just in reverse.

Yeah, although the current kwarg name is `oldest_first`, not `in_reverse`, and if I renamed it to `in_reverse` it would be less clear that it was sorting by age by default. Pretty minor, I know, but given it's not something we're using at the moment it seems easier to leave as-is.